packages/wrangler/src/__tests__/pipelines.test.ts

```diff
@@ -361,6 +412,32 @@ describe("pipelines", () => {
 		expect(requests.count).toEqual(1);
 	});
 
+	it("create - should create a pipeline with auth", async () => {
```

Comment:
should we perhaps group all the `create` specs into a dedicated suite? So 👇 

```
describe("create, () => {
  it("should create a pipeline with auth", async () => {});
  ...
})
```

Same for `update`.

I tend to find this kind of grouping much easier to read, download in my brain, and maintain ^.^

Reply:
Done!  (although, i used to do this, too, but have started going back to defining individual. tests, so that it's easier to isolate individual tests by temporarily adding in `.only` for specific tests that i want to iteratively debug.)

Reply:
oh, does that not work as smoothly with `describe.only` -> `it.only()`?